Sources/Client/Views/RouterView.swift

```diff
@@ -51,16 +48,23 @@ struct RouterView: View {
                   InputView { inputCaptureEnabled, setDelegate in
                     PlayServerView(
                       serverDescriptor: descriptor,
-                      resourcePack: loadedResources.resourcePack,
+                      resourcePack: loadedResources?.resourcePack,
                       inputCaptureEnabled: inputCaptureEnabled,
                       delegateSetter: setDelegate)
                   }
-                case .fatalError(let message):
-                  FatalErrorView(message: message)
-                case .settings:
-                  SettingsView(isInGame: false, eventBus: nil, onDone: {
-                    appState.pop()
-                  })
+                case .settings(let landingPage):
+                /** Simply calling getSettingsView once with the given landingPage doesn't cause States in `SettingsView`
+                    to be properly initialised. A rather reduntant switch statement is needed.
+                 */
+                switch landingPage {
```

Comment:
please change your Xcode setting for switch statement formatting to match my switch style (look at switches elsewhere in delta client)